### DISCUSSION TITLE
Martini hydrated octanol

### DISCUSSION THREAD

**Discussion by EKearn** [Martini-Force-Field-Initiative/MartiniDiscussions#9] [Water]
Hi there,

I noticed in the original Martini 3 paper, simulations to calculate the octanol/water transfer energy used bulk water, but water-saturated octanol. I was wondering why, I can't find where this change originated and how it was setup.

Thanks!
Eoin

**Comment by ricalessandri** [comment]
In the experiments, octanol is in contact with water and so water goes in the octanol phase. We account for this by using water-saturated octanol when computing transfer free energies. Not sure what's the first Martini paper to do that, perhaps already the 2007 JPCB paper.

In any case, if you can't find it in the NatMethods 2021 paper, see the [small molecule paper](https://onlinelibrary.wiley.com/doi/full/10.1002/adts.202100391) for the same procedure:

"Note that we simulate hydrated octanol, that is we add a ≈0.3 mole fraction of water according to experimental conditions.[51]"

[51] L. Bernazzani, S. Cabani, G. Conti, V. Mollica, J. Chem. Soc., Faraday Trans. 1995, 91, 649.
